rich/pretty.py

```diff
@@ -28,8 +29,10 @@
 
 try:
     import attr as _attr_module
+
+    _has_attrs = True
 except ImportError:  # pragma: no cover
-    _attr_module = None  # type: ignore[assignment]
+    _has_attrs = False
```

Comment:
These are always a little painful in MyPy; using this method helped it it out below. I wish mypy had a bit better support try/except and setting to None.